src/storage/heap_file.c

```diff
@@ -10806,13 +10806,6 @@ heap_class_get_partition_info (THREAD_ENTRY * thread_p, const OID * class_oid, O
       return ER_FAILED;
     }
 
-  scan_cache.mvcc_snapshot = logtb_get_mvcc_snapshot (thread_p);
```

Comment:
Why was this source code removed?

Reply:
this code is not necessary because the mvcc snapshot is already built and this deletion issue has been discussed with @wrlawodms enoughly.